pages/common/pio-org.md

```diff
@@ -0,0 +1,28 @@
+# pio org
+
+> Manage PlatformIO organizations and their owners.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/org/>.
+
+- Create a new organization:
+
+`pio org create {{orgname}}`
+
+- Delete an organization:
+
+`pio org destroy {{orgname}}`
+
+- Add a user to an organization:
+
+`pio org add {{orgname}} {{username}}`
+
+- Remove a user from an organization:
+
+`pio org remove {{orgname}} {{username}}`
+
+- List all organizations and their owners, the user owns:
```

Comment:
This command does just what I'm saying in the suggestion, right? That's what the [docs](https://docs.platformio.org/en/latest/core/userguide/org/cmd_list.html#description) say.
```suggestion
- List all organizations and their owners:
```

Reply:
Here too, there are far more organizations, than the user owns and this suggestion would imply that it lists all there are.

Reply:
It doesn't do that?  The docs appear to me to say that.

Reply:
There are many mistakes and bad descriptions in the docs. But at least when I run `pio org list`, I get `You do not have any orginazations`

Reply:
How about this?
```suggestion
- List all organizations the current user is a member of and their owners:
```